odoo/netsvc.py

```diff
@@ -134,6 +134,7 @@ def record_factory(*args, **kwargs):
         'zeep.loader',# zeep using defusedxml.lxml
         'reportlab.lib.rl_safe_eval',# reportlab importing ABC from collections
         'xlrd/xlsx',# xlrd mischecks iter() on trees or something so calls deprecated getiterator() instead of iter()
+        'ofxparse',# reportlab importing ABC from collections
```

Comment:
Maybe with a comment that's correct? And a link to the ofxparse issue (jseutter/ofxparse#155)? And maybe a link to an issue for bullseye so we can track whether / when debian resolves the issue?